web/chromecom.js

```diff
@@ -13,27 +13,31 @@
  * limitations under the License.
  */
 
-/* globals chrome */
+/* globals chrome, DEFAULT_PREFERENCES, DEFAULT_URL */
 'use strict';
 
 (function (root, factory) {
   if (typeof define === 'function' && define.amd) {
     define('pdfjs-web/chromecom', ['exports', 'pdfjs-web/app',
-      'pdfjs-web/overlay_manager', 'pdfjs-web/pdfjs'], factory);
+      'pdfjs-web/overlay_manager', 'pdfjs-web/preferences', 'pdfjs-web/pdfjs'],
+      factory);
   } else if (typeof exports !== 'undefined') {
     factory(exports, require('./app.js'), require('./overlay_manager.js'),
-      require('./pdfjs.js'));
+      require('./preferences.js'), require('./pdfjs.js'));
   } else {
     factory((root.pdfjsWebChromeCom = {}), root.pdfjsWebApp,
-      root.pdfjsWebOverlayManager, root.pdfjsWebPDFJS);
+      root.pdfjsWebOverlayManager, root.pdfjsWebPreferences,
+      root.pdfjsWebPDFJS);
   }
-}(this, function (exports, app, overlayManager, pdfjsLib) {
+}(this, function (exports, app, overlayManager, preferences, pdfjsLib) {
 //#if CHROME
 //#if !CHROME
```

Comment:
This was not introduced in this PR, but looks weird. The nested pre-processor makes this condition always false.
Could you add a comment that explains this weird construct?